roles/ceph-mon/tasks/start_monitor.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: activate monitor with upstart
+  file: path=/var/lib/ceph/mon/ceph-{{ ansible_hostname }}/{{ item }}
+        state=touch
+        owner=root
+        group=root
+        mode=0600
+  with_items:
+    - done
+    - upstart
+  when: ansible_distribution == "Ubuntu"
```

Comment:
I would rather not have flags for ansible_distribution.  All assumptions today are that we are on Ubuntu.  When we get to additional distros, we need to take a more holistic approach to addressing multi-distro patterns.